test/014-request.test.js

```diff
@@ -19,6 +22,27 @@ describe('REQUEST TEST SUITE', () => {
         password: 'johnny777'
       });
     ({ token } = user.body);
+
+    await chai.request(server).post('/api/v1/users')
```

Comment:
You can create this user as an admin so that there's no need to bring in the `User` model for updating here. To make that possible, add `isAdmin` as part of the request body in the `register` controller.

Reply:
ok